src/displayapp/screens/ApplicationList.cpp

```diff
@@ -8,77 +8,75 @@
 
 using namespace Pinetime::Applications::Screens;
 
-ApplicationList::ApplicationList(Pinetime::Applications::DisplayApp *app) :
-        Screen(app),
-        screens{app, {
-                [this]() -> std::unique_ptr<Screen> { return CreateScreen1(); },
-                [this]() -> std::unique_ptr<Screen> { return CreateScreen2(); },
-                //[this]() -> std::unique_ptr<Screen> { return CreateScreen3(); }
-          }
-        } {}
-
+ApplicationList::ApplicationList(Pinetime::Applications::DisplayApp *app) : Screen(app),
+                                                                            screens{app, {
+                                                                                             [this]() -> std::unique_ptr<Screen> { return CreateScreen1(); }, [this]() -> std::unique_ptr<Screen> { return CreateScreen2(); },
+                                                                                             //[this]() -> std::unique_ptr<Screen> { return CreateScreen3(); }
+                                                                                         }}
+{
+}
```

Comment:
The changes in this file is just code formating right? I'm pretty sure the previous formatting is preferred in this project.

Reply:
in so many files something escaped ...